gufe/visualization/mapping_visualization.py

```diff
@@ -305,3 +307,57 @@ def draw_unhighlighted_molecule(mol, d2d=None):
         bond_colors=[{}],
         highlight_color=red,
     )
+
+
+def _concat_images(imgs, buffer_spacing=4, ncols=3)-> Image:
```

Comment:
I agree, this PR is a very old one and I wanted to bring it to your attention as I think it could be a handy feature. 

hm... I wonder if this function should be used in line 360 somehow? I also wonder if the code developed away from the initial implementation?